linera-base/src/crypto/signer.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+
+use super::CryptoHash;
+use crate::{
+    crypto::{AccountPublicKey, AccountSignature, BcsSignable},
+    identifiers::AccountOwner,
+};
+
+/// Wrapper around bytes that can be signed.
+#[derive(Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct SignableBytes(Vec<u8>);
+impl SignableBytes {
+    /// Creates a new `SignableBytes` from the given bytes.
+    pub fn new(bytes: Vec<u8>) -> Self {
+        SignableBytes(bytes)
+    }
+
+    /// Returns the inner bytes.
+    pub fn as_bytes(&self) -> &[u8] {
+        &self.0
+    }
+}
+impl BcsSignable<'_> for SignableBytes {}
```

Comment:
What is this for? This is defeating the purpose of `BcsSignable`.

Reply:
Yes, this should be removed now - i.e. the whole `SignableBytes` struct.

Reply:
Removed.